homework/thinhbka/lesson-4-hashing/4_Insert_Delete_GetRandom_O(1).py

```diff
@@ -0,0 +1,44 @@
+class RandomizedSet:
+
+    def __init__(self):
+        """
+        Initialize your data structure here.
+        """
+        self.dic={}
+        self.count=0
+
+    def insert(self, val: int) -> bool:
+        """
+        Inserts a value to the set. Returns true if the set did not already contain the specified element.
+        """
+        if val in self.dic:
+            return False
+        self.dic[val] = val
+        self.count+=1
+        # print("insert success", self.dic)
+        return True
+      
+    def remove(self, val: int) -> bool:
+        """
+        Removes a value from the set. Returns true if the set contained the specified element.
+        """
+        
+        if val in self.dic:
+            del self.dic[val]
+            return True
+        return False
+
+    def getRandom(self) -> int:
+        """
+        Get a random element from the set.
+        """
+     
+        return random.choice(list(self.dic.values()))
```

Comment:
will it cost O(n)?

Reply:
Hi, @quanghoang33. Although this solution still AC in LC, but it is definitely take O(n) for get random. 
I will change this code and PR latter.
Regards

Reply:
Changed